bundler.d/windows.rb

```diff
@@ -1,5 +1,5 @@
 group :windows do
   gem 'highline', :platform => :mingw
   gem 'win32-service', :platforms => :mingw
-  gem "win32-open3", :platforms => :mingw_18
+  gem 'wopen3', '>= 0.3', :platforms => :mingw
```

Comment:
I haven't tested smart-proxy with wopen3 gem, and I don't think there's a need for it: modern rubies' implementation of Open3 library is good enough for us.